checklists/network_appdelivery_checklist.en.json

```diff
@@ -83,7 +83,7 @@
             "id": "A01.08",
             "severity": "Medium",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
-            "link": "https://learn.microsoft.com/azure/web-application-firewall/ag/ag-overview"
+            "link": "https://learn.microsoft.com/en-us/azure/application-gateway/tutorial-protect-application-gateway-ddos"
```

Comment:
Could you please remove localization (delete the `/en-us` part of the URL) from the link?
Thanks!

Reply:
Sure, will do. Thanks Jose!